ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -303,6 +303,9 @@ Rectangle {
                 event.accepted = true
                 messageTooLongDialog.open()
             }
+        } else if (event.key === Qt.Key_Escape && control.isReply) {
+            control.isReply = false
```

Comment:
```suggestion
            root.isReply = false
```
:pray:

Reply:
I'd prefer we did not introduce the `root` change here as this is very error prone and we risk introducing new issues to the chat input.

Let's have the fixes for this PR land first, then refactor the component if necessary in another, dedicated, step

Reply:
That's fair. That's why I approved this PR and only asked ("prayed") for the change.

I don't see it being very error-prone, it is simple find-and-replace most of the time unless `control` id is used in another, tightly-coupled component. Anyway, the risk of regression exists and I understand the benefits of having very isolated PR.